src/librustc_mir/build/expr/as_constant.rs

```diff
@@ -18,7 +18,7 @@ impl<'a,'tcx> Builder<'a,'tcx> {
     /// Compile `expr`, yielding a compile-time constant. Assumes that
     /// `expr` is a valid compile-time constant!
     pub fn as_constant<M>(&mut self, expr: M) -> Constant<'tcx>
-        where M: Mirror<'tcx, Output=Expr<'tcx>>
+        where M: Mirror<'tcx, Output = Expr<'tcx>>
```

Comment:
I do, though it took a little getting used to. We should at least have an option for this.